packages/neuron-ui/src/styles/mixin.scss

```diff
@@ -24,7 +24,7 @@
   border: none;
   border-radius: 16px;
   background-color: var(--secondary-background-color);
-  min-width: 450px;
+  min-width: 360px;
```

Comment:
Have you made a double-check in the other page?

Reply:
checked